preview/test/widget.spec.ts

```diff
@@ -52,6 +52,23 @@ test.describe('Preview widget', () => {
         await expect(page).toHaveScreenshot('widget-expanded.png');
     });
 
+    test('should not display the experience if not specified', async ({page}) => {
+        await open(page, {
+            audience: 'A very very very very long audience name',
```

Comment:
I guess this should follow the 'Unknown' example, since the user will see "None" instead of a long audience

```suggestion
            audience: 'None',
```

Reply:
I would prefer to keep it following the other tests. I understand your idea, the experience won't be shown only in the "none audience" case. But the widget doesn't know the audience's name, it just shows what was received in the params.